### DISCUSSION TITLE
Harvester loaded 1 plot, but 0 plots were eligible for farming

### DISCUSSION THREAD

**Discussion by hhy80** [Chia-Network/chia-blockchain#1439] [Q&A]
I started chia node, farmer, harvester on a same machine with Ubuntu 18.04 LTS.

After created a plot, I found everything worked well except Harvester. The INFO were printed below:

```
Searching directories ['/home/chia/data', '/home/chia/final_data']
13:46:38.746 harvester src.plotting.plot_tools : INFO     Loaded a total of 1 plots of size 0.09892603841763048 TiB, in 0.01839303970336914 seconds
13:46:38.746 harvester src.harvester.harvester : INFO     0 plots were eligible for farming e87ec7ba25... Found 0 proofs. Time: 0.01875 s. Total 1 plots
```

Cooould you pls give some hints to fix it ?

**Comment by hoffmang9** [comment]
only plots of size k=32 or larger are eligible on mainnet.

**Reply by hhy80** [reply]
Yes, I used k=32 for plot.  I printed the farm summary here.  101.300GiB is corresponded to k=32.

```
Farming status: Farming
Total chia farmed: 0.0
User transaction fees: 0.0
Block rewards: 0.0
Last height farmed: 0
Plot count: 1
Total size of plots: 101.300 GiB
Estimated network space: 131.444 PiB
Expected time to win: 275 days
```

**Comment by hoffmang9** [comment]
`Loaded a total of 1 plots of size 0.09892603841763048 TiB`

**Reply by hhy80** [reply]
I used the command to create a plot, is that correct?

`chia plots create -k 32 -b 4000 -t /home/chia/temp_data -d /home/chia/final_data`

**Reply by trepca** [reply]
I have the same problem, it works if it's running a full node, but not on harvester.

**Reply by hhy80** [reply]
> I have the same problem, it works if it's running a full node, but not on harvester.

I'm trying to create a larger plot, hope it can work.

So you're able to farming on the plot with the same method to create? Any XCH you got?

**Reply by qukuailianyonghu** [reply]
i am meeting the same problem, do you solved this issue?

**Reply by trepca** [reply]
it's not an issue, it's just an optimisation feature to filter out plots that are not even close to challenge.

**Reply by Sagittarius** [reply]
I got the same situation. So how to be sure harvester is working? I got have 2  machines in my home network, one is debian linux, another is windows 10, and configure them follow offical guide(https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines).I make debian as main machine, make windows as harvester.
But I got these logs as above mentioned:
```
09:21:02.080 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 1399...... Found 0 proofs. Time: 0.19754 s. Total 13 plots
09:21:02.081 harvester harvester_server        : INFO     -> farming_info to peer 192.168.50.62 8fe5...blablabla...9c00c3a
```
Is it normal?
If I copy these plots to debian, I can see new plots under debian farming.
Thx

**Reply by szmikezhang** [reply]
on harvester  use command chia keys add , input 24 mnemonic with main machine ,first create wallet

**Reply by Sagittarius** [reply]
感觉中国人多啊，可以说中文了。 就说这个日志是不是正常的，现在我发现会有几条日志不一样了：
`2 plots were eligible for farming f9ca08d43d... Found 0 proofs. Time: 0.33536 s. Total 10 plots`
但是仍然是`0 proofs`
